extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -524,7 +524,13 @@ export class JupyterServerInstallation {
 
 		let packagesResult: PythonPkgDetails[] = [];
 		if (packagesInfo) {
-			packagesResult = <PythonPkgDetails[]>JSON.parse(packagesInfo);
+			try {
+				packagesResult = <PythonPkgDetails[]>JSON.parse(packagesInfo);
```

Comment:
FYI - we have an error-handling JSON parser that we can look into using instead of JSON.parse for future reference. I think it's JSONC package. You should look into using this too, but OK to do in separate less high-pri review.